adapters/handlers/graphql/local/get/get_test.go

```diff
@@ -324,8 +325,8 @@ func TestExtractAdditionalFields(t *testing.T) {
 			},
 			resolverReturn: []interface{}{
 				map[string]interface{}{
-					"_additional": &models.AdditionalProperties{
-						Classification: &models.AdditionalPropertiesClassification{
+					"_additional": models.AdditionalProperties{
+						"classification": &additional.AdditionalPropertiesClassification{
```

Comment:
Could this just be `additional.Classification`?

Reply:
I have renamed this struct to `additional.Classification`